src/components/core/DpEditor/DpEditor.vue

```diff
@@ -1056,7 +858,7 @@ export default {
       new Heading({ levels: this.toolbar.headings })
     ]
 
-    if (this.toolbar.boilerPlate) {
+    if (this.editorInsertAtCursorPos) {
```

Comment:
Would it do any harm to always add the `EditorInsertAtCursorPos` extension? If not, we could drop the `editorInsertAtCursorPos` prop entirely and have a cleaner component API.